sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -44,3 +44,12 @@ export class UpdateStudentContactDto {
   @IsNotEmpty()
   postalCode: string;
 }
+
+export class ProgramYearDto {
```

Comment:
The DTO validations don't apply for response DTOs.
Please, remove the decorators and convert the class to an interface.